descartes_planner/include/descartes_planner/planning_graph.h

```diff
@@ -144,17 +146,17 @@ class PlanningGraph
   //       and include an accessor to both formats
 
   // maintains the original (Cartesian) points list along with link information and associated joint trajectories per point
-  std::map<descartes_core::TrajectoryPt::ID, CartesianPointInformation> *cartesian_point_link_;
+  CartesianMap* cartesian_point_link_;
```

Comment:
Well, I intend to re-factor a lot of things going forward. That use of heap allocation is one. I can add it to this one, but there will be a continuous stream of these PR's fixing small things.